scrapers/va/csv_bills.py

```diff
@@ -9,38 +9,10 @@
 import time
 
 from .common import SESSION_SITE_IDS, COMBINED_SESSIONS
+from .actions import Categorizer
 
 tz = pytz.timezone("America/New_York")
 SKIP = "~~~SKIP~~~"
```

Comment:
Dup of `SKIP` above in actions.py?

Reply:
yes! and actually unnecessary in both places - will delete, thank you